static/app/views/explore/components/suspectTags/charts.tsx

```diff
@@ -22,25 +31,191 @@ export function Charts({rankedAttributes}: Props) {
   );
 }
 
+type CohortData = SuspectAttributesResult['rankedAttributes'][number]['cohort1'];
+
+function cohortsToSeriesData(
+  cohort1: CohortData,
+  cohort2: CohortData
+): {
+  [BASELINE_SERIES_NAME]: Array<{label: string; value: string}>;
+  [SELECTED_SERIES_NAME]: Array<{label: string; value: string}>;
+} {
+  const cohort1Map = new Map(cohort1.map(({label, value}) => [label, value]));
+  const cohort2Map = new Map(cohort2.map(({label, value}) => [label, value]));
```

Comment:
Curious, is there a reason you used a map here instead of an object? It looks like your just using `cohort1Map` as a simple key value mapping and not taking advantage of the benefits of a map (like that it's iterable and ordered)